blog/resoto-search-aggregation/index.md

```diff
@@ -0,0 +1,127 @@
+---
+authors: [matthias]
+tags: [search, aws, gcp]
+---
+
+# Aggregating Data with Resoto
```

Comment:
```suggestion
# Aggregating Search Data
```
I was thinking about this, and don't think we need `Resoto` in each of these blog post titles... after all, it _is_ the Resoto blog.